pennylane_forest/device.py

```diff
@@ -244,21 +243,36 @@ def apply(self, operations, **kwargs):
 
         # Apply the circuit operations
         for i, operation in enumerate(operations):
-            # number of wires on device
+            # map the operation wires to the physical device qubits
             wires = self.remap_wires(operation.wires)
             par = operation.parameters
 
             if i > 0 and operation.name in ("QubitStateVector", "BasisState"):
-                raise DeviceError("Operation {} cannot be used after other Operations have already been applied "
-                                  "on a {} device.".format(operation.name, self.short_name))
-
+                raise DeviceError("Operation {} cannot be used after other Operations have already "
+                                  "been applied on a {} device.".format(operation.name, self.short_name))
             self.prog += self._operation_map[operation.name](*par, *wires)
 
-        # Apply the circuit rotations
+        self.prog += self.apply_rotations(rotations)
+
+    def apply_rotations(self, rotations):
+        """Apply the circuit rotations.
+
+        This method serves as an auxiliary method to :meth:`~.ForestDevice.apply`.
+
+        Args:
+            rotations (List[pennylane.Operation]): operations that rotate into the
+                measurement basis
+
+        Returns:
+            list: wires as integers corresponding to the wiring if applicable
```

Comment:
doesn't this method return a pyQuil program?

Reply:
Oh you're right, thanks for that!